src/components/chart/ha-chart-base.ts

```diff
@@ -470,7 +495,6 @@ export class HaChartBase extends LitElement {
       tooltips.forEach((tooltip) => {
         tooltip.confine = true;
         tooltip.appendTo = undefined;
-        tooltip.triggerOn = "click";
```

Comment:
This was added on purpose. See #24083

Reply:
This one I guess is tricky as my intent clashes with the reason why it was added in the first place.